### PR TITLE
release notes for 2.11.1.0

### PR DESCRIPTION
**Pull request by dhrubo-os**
### Description
[release notes for 2.11.1.0]
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-ml-common.release-notes-2.11.1.0.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Compatible with OpenSearch 2.11.1
+ 
+ 
+### Enhancements
+ 
+ 
+### Infrastructure
+ 
+ 
+### Bug Fixes
```
[the remark this answers is not in the document]
Comment (dhrubo-os) [reply release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:12]: Addressed.

File: release-notes/opensearch-ml-common.release-notes-2.11.1.0.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+## Version 2.11.0.0 Release Notes
+ 
+Compatible with OpenSearch 2.11.1
+ 
+ 
+### Enhancements
```
Comment (ylwu-amzn) [inline release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:6]: Should we keep this part if no item?

Comment (dhrubo-os) [reply release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:6]: Isn't that same for other cases, like `Documentation`, `Maintenance`, `Refactoring`?

Comment (dhrubo-os) [reply release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:6]: Do you want me to remove other empty headers too?

Comment (ylwu-amzn) [reply release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:6]: yes, I think we should remove all other empty headers too

File: release-notes/opensearch-ml-common.release-notes-2.11.1.0.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+## Version 2.11.0.0 Release Notes
```
Comment (HenryL27) [inline release-notes/opensearch-ml-common.release-notes-2.11.1.0.md:1]: should this say 2.11.1.0?

Comment (dhrubo-os) [reply release-notes/opensearch-ml-common.release-notes-2.11.1.0.md]: Good catch. Addressed.

**Review by ylwu-amzn — APPROVED** [review]

**Review by b4sjoo — APPROVED** [review]

**Pull request merged**